data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -1,166 +1,264 @@
 #textdomain wesnoth-ei
+
+#
+# if the player is already struggling, this scenario is easy. With no recall list, you can win by just rushing Gweddry/Owaec/Dacyn acrosss
+# and if you have a single castle of worthwhile recalls, it's still fairly easy to get across
+#
+# but a veteran player with a huge recall list will have difficulty getting everyone across the bridge safely, because of both the gold costs and the time limit
+# this is designed to equalize things a little bit, and make it a little harder for a player's recall list to snowball out of control
+#
+
+#
+# the extremely low turn limit is to discrouage winning via killing enemy leaders.
+# while killing enemy leaders is an option, it bypasses the ending cutscene and feels against the spirit of the scenario
+# instead, players are encouraged to rush across the bridge as quickly as possible, which takes far fewer turns than direct combat
+#
+# low turn limit also discourages using the bridge chokepoint to farm excessive kills
+#
+# could always disable "winning via killing enemy leaders" completely, but I feel like there's no in-universe reason for that
```

Comment:
So "we're overwhelmingly outnumbered" is not an in-universe reason?

Reply:
As in, the player kills all the enemy leaders and then a bunch more enemies spawn to chase you away? That could work, though for thematic reasons I'd still prefer keeping the turn limit low on this one (if that's your reason for commenting).